src/styles/blocks/item.scss

```diff
@@ -0,0 +1,89 @@
+.item {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px;
+
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  font-family: ' Roboto ', sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
```

Comment:
it's default value - no need to set it

Reply:
It`s not default

Reply:
It's just a fact) Some font-weight has to be default: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)

![image](https://github.com/user-attachments/assets/797906b5-5087-4eaa-aa9a-11a7051cb8fe)